test/browser/component-utils/controlled-app.js

```diff
@@ -1,32 +1,33 @@
 /* eslint-env mocha */
 /* global expect */
 
+import requestAnimationFrameCB from 'raf';
```

Comment:
What's the reason for using a polyfill rather than the browser's `requestAnimationFrame`?

Reply:
raf handles both node and browser. It also does polyfilling for different browsers like mozRequestAnimationFrame. I think now most browsers are consistent so may be we don't need it.

Reply:
Yeah I don't think we need it here. These are browser-only tests (Node tests are in `test/server/`), and every browser we support/test on has a working `requestAnimationFrame` implementation (as seen by the liberal use of it in the existing browser tests).